### PR TITLE
doc/orchestrator: update rgw creation

### PR DESCRIPTION
**Pull request by yehudasa**
Signed-off-by: Yehuda Sadeh <yehuda@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/bootstrap.rst [hunk 1]
```diff
@@ -162,14 +162,32 @@ Deploying RGWs
...
+zones, see :ref:`multisite`.)
 
 Note that with cephadm, radosgw daemons are configured via the monitor
 configuration database instead of via a `ceph.conf` or the command line.  If
 that confiruation isn't already in place (usually in the
 ``client.rgw.<realmname>.<zonename>`` section), then the radosgw
 daemons will start up with default settings (e.g., binding to port
 80).
+
+If a realm has not been created yet, then the first step is to create a new realm,::
```
Comment (liewegas) [inline doc/bootstrap.rst]: ..., first create a new realm::

File: doc/bootstrap.rst [hunk 2]
```diff
@@ -162,14 +162,32 @@ Deploying RGWs
...
+
+  # radosgw-admin realm create --rgw-realm=<realm-name> --default
+
+The next step would be to create a new zonegroup,::
```
Comment (liewegas) [inline doc/bootstrap.rst]: Next, create a new zonegroup::

File: doc/bootstrap.rst [hunk 3]
```diff
@@ -162,14 +162,32 @@ Deploying RGWs
...
+
+  # radosgw-admin zonegroup create --rgw-zonegroup=<zonegroup-name>  --master --default
+
+And then a new zone needs to be created,::
```
Comment (liewegas) [inline doc/bootstrap.rst]: Next, create a zone::

File: doc/bootstrap.rst (hunk 2 above)
Comment (liewegas) [reply doc/bootstrap.rst]: Does it make sense to suggest names for the zonegroup? I think in the absence of any context, it's not obvious what sort of name to choose.  Maybe "myorg" for realm, "default" for zonegroup, and "us-east-1" for zone, or something?

Comment (yehudasa) [reply doc/bootstrap.rst]: Can do these. Should we add an example for all of these commands? The one example at the end doesn't have a zonegroup.

**Comment by sebastian-philipp** [comment]
I totally missed this PR. sorry!

File: doc/bootstrap.rst [hunk 4]
```diff
@@ -162,14 +162,35 @@ Deploying RGWs
...
+
+  # ceph orch apply rgw *<realm-name>* *<zone-name>* *<num-daemons>* [*<host1>* ...]
+
+For example, to deploy 2 rgw daemons serving the *myorg* realm and the *us-east-1*
+zone on *myhost1* and *myhost2*::
+
+  # radosgw-admin realm create --rgw-realm=myorg --default
+  # radosgw-admin zonegroup create --rgw-zonegroup=default --master --default
+  # radosgw-admin zone create --rgw-zonegroup=default --rgw-zone=us-east-1 --master --default
+  # ceph orch apply rgw myorg us-east-1 2 myhost1 myhost2
```
Comment (sebastian-philipp) [inline doc/bootstrap.rst]: in a way, this looks like a bug of cephadm. There should be no need to acually make those calls by hand.

**Comment by sebastian-philipp** [comment]
ping?

**Comment by yehudasa** [comment]
@sebastian-philipp @liewegas rebased

File: doc/bootstrap.rst (hunk 4 above)
Comment (yehudasa) [reply doc/bootstrap.rst]: @sebastian-philipp probably, but at the moment that's the way it can be done.

**Comment by liewegas** [comment]
can you retarget/rebase this at octopus branch?

**Comment by yehudasa** [comment]
@liewegas rebased and re-targeted.

**Pull request merged**